lib/sprockets/sass/utils.rb

```diff
@@ -57,6 +57,12 @@ def get_class_by_version(class_name, version = version_of_sprockets)
           nil
         end
 
+        def get_less_class_by_version(class_name, version = version_of_sprockets)
+          constantize("Sprockets::Less::V#{version}::#{class_name}")
```

Comment:
Maybe would be better to check first ``` if defined?(Sprockets::Less) ``` before trying to constantize the string and just return nil, which will be handled in the Importer  , where we actually call try to add this class in the list of processors for that file.